### ISSUE TITLE
[Proposal] Server app needs optimization

### ISSUE THREAD

**Issue by Hzfinfdu** [OpenMOSS/Language-Model-SAEs#13]
I have noticed 2 problems in current frontend + backend service.

- [ ] [urgent] byte_decoder does not work in llama-3 8B decoder. What is a workaround?

AttributeError: 'PreTrainedTokenizerFast' object has no attribute 'byte_decoder'.

- [ ] [low priority] Larger models have longer context window. Will this make visualization messier?

Simple truncating might not work. It may fail to catch longer range dependency.

I propose to make some changes in the frontend. What about a preview (shorter local context) which can be expanded to full context? @dest1n1s

**Comment by dest1n1s** [comment]
I also agree with a preview, which marks out the top activation in a context. It can be expanded to full context when clicking the preview.

**Comment by dest1n1s** [comment]
Should already be supported with #39 and #48.
